mongo/session.go

```diff
@@ -89,7 +120,7 @@ func (s *sessionImpl) ID() bsonx.Doc {
 	return s.clientSession.SessionID
 }
 
-// EndSession ends the session.
+// EndSession implements the Session interface.
```

Comment:
Can't comment above, but on line 113, what is XSession?  Should that just be Session?

Reply:
`XSession` is an interface we added to be able to retrieve the underlying client session (which is in `x`). This was necessary for some of the spec tests, which make assertions about the state of the pinned server for sharded transactions.

Defined at https://github.com/mongodb/mongo-go-driver/blob/master/mongo/session.go#L64.